index.html

```diff
@@ -252,17 +252,17 @@ <h1>About <br />Myself</h1>
       </ul>
     </section>
     <footer class="footer">
-      <form action="" method="post">
-          <h1 class="form_header">Contact me</h1>
-          <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-              
-          <div class="form_inputs">
-              <input type="text" placeholder="Name" class="form_input" maxlength="30" required class="form_input">
```

Comment:
Nice work though, but i think it is better not to leave any blank lines in your code where unnecessary.  Doing this your code looks your code unclear.
Kindly please remove that blank line. Thank you